blueprints/app/files/babel.config.js

```diff
@@ -0,0 +1,17 @@
+const { buildEmberPlugins } = require('ember-cli-babel');
+
+module.exports = function (api) {
+  api.cache(true);
+
+  return {
+    presets: [
+      [
+        require.resolve('@babel/preset-env'),
+        {
+          targets: require('./config/targets'),
```

Comment:
This require is failing when running inside an addon (because the file is in a different location: `tests/dummy/config/targets.js`)

Reply:
Fixed in #9556. I had a problem with this branch and had to re-create the PR.